vpr/src/pack/cluster.cpp

```diff
@@ -2128,13 +2149,23 @@ static void update_cluster_stats(const t_pack_molecule* molecule,
         VTR_ASSERT(atom_pb);
 
         cur_pb = atom_pb->parent_pb;
+
+        //Update attraction group
+        AttractGroupId atom_grp_id = attraction_info.get_atom_attraction_group(blk_id);
+
         while (cur_pb) {
             /* reset list of feasible blocks */
             if (cur_pb->is_root()) {
                 cb = cur_pb;
             }
             cur_pb->pb_stats->num_feasible_blocks = NOT_VALID;
             cur_pb->pb_stats->num_child_blocks_in_pb++;
+
+            if (atom_grp_id != NO_ATTRACTION_GROUP) {
```

Comment:
The StrongId types (like AttractGroupId) will have AttractGroupId::INVALID(). Use that and get rid of NO_ATTRACTION_GROUP